src/foam/nanos/auth/email/EmailVerificationCode.js

```diff
@@ -61,6 +61,22 @@ foam.CLASS({
       class: 'DateTime',
       name: 'expiry',
       hidden: true
+    },
+    {
+      class: 'Boolean',
+      name: 'processed',
```

Comment:
Why do we mark it as processed rather than just removing it?

Reply:
Fixed in https://github.com/kgrgreer/foam3/pull/2287/commits/ae74483c87036e54e19c6f3ae6920781851a96c4.